runtime/common/src/constants.rs

```diff
@@ -275,3 +275,8 @@ impl Clone for MessagesMaxPayloadSizeBytes {
 pub type MSAMortalityWindowSize = ConstU32<100>;
 pub type MSAMaxSignaturesPerBucket = ConstU32<50_000>;
 pub type MSANumberOfBuckets = ConstU32<2>;
+
+parameter_types! {
+	/// SS58 Prefix for the for Frequency Network
+	pub const Ss58Prefix: u16 = prod_or_testnet_or_local_or_env!(90, 42, 42);
+}
```

Comment:
SS58 prefix 90 for prod  42 and 42 for rococo local and testnet

Reply:
Using the macro to select

Reply:
works for me